worlds/ladx/LADXR/generator.py

```diff
@@ -153,7 +153,7 @@ def generateRom(args, world: "LinksAwakeningWorld"):
         patches.witch.updateWitch(rom)
     patches.softlock.fixAll(rom)
     patches.maptweaks.tweakMap(rom)
-    if world.options.overworld == "open_mabe":
+    if world.ladxr_settings.overworld == "openmabe":

```

Comment:
Is it intended that the string changed from open_mabe to openmabe?

Reply:
Yeah, LADXR settings have no delimiters, see how I read it in the logic file. I switched it because it's a better fit for existing convention to use the LADXR setting when available.